src/util/persistent_logger.js

```diff
@@ -216,7 +216,7 @@ class PersistentLogger {
 
     async _open() {
         await native_fs_utils._create_path(this.dir, this.fs_context);
-        return nb_native().fs.open(this.fs_context, this.active_path, 'as');
+        return nb_native().fs.open(this.fs_context, this.active_path, 'as+');
```

Comment:
how does a and + work together? append with rw fits?

Reply:
I think so. I tested it and it seemed to work just fine. I relied on [this](https://stackoverflow.com/a/50875791) stackoverflow answer.